katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/AbstractQueryExecutorImpl.java

```diff
@@ -95,8 +95,7 @@ public MetaDataObject getMeta() {
 				entityList.add((T) values[0]);
```

Comment:
this whole function `getResultList()` can be reduced to 
`return isCompoundSelection() ? list.stream().map(Object[].class::cast).map(o -> o[0]).collect(toList()) : (List<T>)list;`